docs/content/deployment/executors.mdx

```diff
@@ -22,6 +22,24 @@ Every job has an executor. The default executor is the <PyObject object="multi_o
 
 An executor can be specified directly on the job by supplying an <PyObject object="ExecutorDefinition" /> to the `executor_def` parameter of <PyObject object="job" decorator /> or <PyObject object="GraphDefinition" method="to_job" />.
 
+```python file=/deploying/executors/executors.py startafter=start_executor_on_job endbefore=end_executor_on_job
+from dagster import multiprocess_executor, job, graph
+
+# Providing an executor using the job decorator
+@job(executor_def=multiprocess_executor)
+def the_job():
+    ...
+
+
+@graph
+def the_graph():
+    ...
+
+
+# Providing an executor using graph_def.to_job(...)
+other_job = the_graph.to_job(executor_def=multiprocess_executor)
+```
+
 A default executor can be specified for all jobs and assets provided to a repository using the `default_executor_def` argument of <PyObject object="repository" decorator />. All jobs that don't specify an executor will use this default executor, but if a job explicitly specifies an executor, then the default provided to the repository will not be used.
```

Comment:
while you're adding code snippets, an example of this could also be helpful

Reply:
whoops I did, just forgot to reference it in the markdown.